gettsim/data/arbeitsl_geld_2.yaml

```diff
@@ -1,730 +1,274 @@
+    5: 251
+    6: 215
+    reference: G. v. 24.03.2011 BGBl. I S. 453.
+  2012-01-01:
+    1: 374
+    2: 337
+    3: 299
+    4: 287
+    5: 251
+    6: 219
+    reference: V. v. 20.10.2011 BGBl. I S. 2093.
+  2013-01-01:
+    1: 382
+    2: 345
+    3: 306
+    4: 289
+    5: 255
+    6: 224
+    reference: B. v. 18.10.2012 BGBl. I S. 2175.
+  2014-01-01:
+    1: 391
+    2: 353
+    3: 313
+    4: 296
+    5: 261
+    6: 229
+    reference: B. v. 16.10.2013 BGBl. I S. 3857.
+  2015-01-01:
+    1: 399
+    2: 360
+    3: 320
+    4: 302
+    5: 267
+    6: 234
+    reference: B. v. 15.10.2014 BGBl. I S. 1620.
+  2016-01-01:
+    1: 404
+    2: 364
+    3: 324
+    4: 306
+    5: 270
+    6: 237
+    reference: B. v. 22.10.2015 BGBl. I S. 1792.
+  2017-01-01:
+    1: 409
+    2: 368
+    3: 327
+    4: 311
+    5: 291
+    6: 236
+    reference: G. v. 22.12.2016 BGBl. I S. 3159.
+  2018-01-01:
+    1: 416
+    2: 374
+    3: 332
+    4: 316
+    5: 296
+    6: 240
+    reference: V. v. 08.11.2017 BGBl. I S. 3767.
+  2019-01-01:
+    1: 424
+    2: 382
+    3: 339
+    4: 322
+    5: 302
+    6: 245
+    reference: V. v. 19.10.2018 BGBl. I S. 1766.
+  2020-01-01:
+    1: 432
+    2: 389
+    3: 345
+    4: 328
+    5: 308
+    6: 250
+    reference: V. v. 15.10.2019 BGBl. I S. 1452.
+
+anteil_regelsatz:
+  name:
+    de: Anteil am Regelsatz für weitere Personen im Haushalt
+    en: Fraction of standard rate
+  description:
+    de: § 20 (3) SGB II, eigentliche Wirknorm ist § 20 IV SGB II i.V.m Absatz V derselben Norm. 2005 bis 2010.
+    en:
+  note:
+    de: 2005-2010 der Regelsatz für weitere Haushaltsmitglieder wurden als Anteil des Standardsatzes bestimmt.
+    en:
+  unit: share
+  2005-01-01:
+    zwei_erwachsene: 0.9
+    weitere_erwachsene: 0.8
+    kinder_0_6: 0.6
+    kinder_7_13: 0.6
+    kinder_14_24: 0.8
+    reference: G. v. 24.12.2003 BGBl. I S. 2954.
+  2009-07-01:
+    deviation_from: previous
+    reference: G. v. 02.03.2009 BGBl. I S. 416.
+    kinder_7_13: 0.7
+
+mehrbedarf_anteil:
+  name:
+    de_1: Mehrbedarf AE, mind. einem Kind wenn Anspruch größer als nach für Kinder unter 7
+    de_2: Mehrbedarf AE, mind. einem Kind unter 7 Jahren oder zwei oder drei Kindern unter 16 Jahren
+    de_3: Maximalanteil Mehrbedarf für Alleinerziehende gemessen am maßgebenden Regelbedarf
+    en_1: Additional need, single parent, one or more children
+    en_2: Additional need, single parent, one child below 7 or two or three children below 16
+    en_3: Upper limit additional need for single parents
+  description:
+    de: § 21 (3) Nr. 1 SGB II.
+    en:
+  unit: share
+  time: month
+  2005-01-01:
+    min_1_kind: 0.12
+    kind_unter_7_oder_mehr: 0.36
+    max: 0.6
+    reference: G. v. 24.12.2003 BGBl. I S. 2954.
+
+vermögensfreibetrag:
+  name:
+    de_1: Vermögensgrundfreibetrag je Lebensjahr, nach Geburtsjahr, erst ab 2008 unterschiedlich
+    de_2: Vermögensfreibetrag für Erstausstattung
+    de_3: Vermögensgrundfreibetrag je Kind
```

Comment:
the mapping of `_1, _2, _3` to the sub keys is wrong or not clear (neither for the 2005-01-01 entry nor for the following).

Reply:
Is not ment to be a mapping! See above.